WcaOnRails/app/controllers/static_pages_controller.rb

```diff
@@ -159,6 +159,21 @@ def score_tools
   def logo
   end
 
+  def teams_committees
+    # get all users who hold one or more officer positions
+    officer_users = Team.all_officers.map(&:current_members).inject(&:+).map(&:user)
+    treasurers = Team.wfc.current_members.filter(&:team_leader).map(&:user)
```

Comment:
This line is failing at stagin with

410374b]   CACHE TeamMember Load (0.0ms)  SELECT  `team_members`.* FROM `team_members` WHERE `team_members`.`team_id` = 6 AND (end_date IS NULL OR end_date > '2019-04-20') LIMIT 11
F, [2019-04-20T00:02:26.322810 #13050] FATAL -- : [7e2e1829-396c-486c-a7ea-1a591410374b] NoMethodError (undefined method `filter' for #<TeamMember::ActiveRecord_Associations_CollectionProxy:0x000000000936bd80>):
F, [2019-04-20T00:02:26.322838 #13050] FATAL -- : [7e2e1829-396c-486c-a7ea-1a591410374b]   
F, [2019-04-20T00:02:26.322858 #13050] FATAL -- : [7e2e1829-396c-486c-a7ea-1a591410374b] app/controllers/static_pages_controller.rb:165:in `teams_committees'
[7e2e1829-396c-486c-a7ea-1a591410374b] lib/middlewares/warden_user_logger.rb:13:in `call'
[7e2e1829-396c-486c-a7ea-1a591410374b] lib/middlewares/fix_accept_header.rb:14:in `call'

Reply:
Indeed, it should be `select` =)

Reply:
@UnsolvedCypher Could you fix this? I think it's the last thing, apart from rebasing.

Reply:
@pedrosino yes, this is all set now :)

Reply:
Actually there are two other occurrences just below, which are still failing.

Reply:
Thank you, I just fixed those